src/docs/deployment/android.md

```diff
@@ -444,9 +394,10 @@ as described in [build an APK](#build-an-apk) using the
 ### What are the supported target architectures?
 
 When building your application in release mode,
-Flutter apps can be compiled for [armeabi-v7a][] (32-bit)
-and [arm64-v8a][] (64-bit). Flutter does not currently support
-building for x86 Android (See [Issue 9253][]).
+Flutter apps can be compiled for [armeabi-v7a][] (ARM 32-bit),
+[arm64-v8a][] (ARM 64-bit) and [x86-64][] (x86 64-bit).
+Flutter does not currently support building for x86 Android
```

Comment:
"currently" implies that we do plan to support it. If that's not true, I'd just delete "currently".

Reply:
Yup. We might in the future, but it's not a priority at the moment.